connectors/sources/github.py

```diff
@@ -231,47 +237,47 @@ class GithubQuery(Enum):
             title
             body
             state
-            assignees(first: 100) {
-            pageInfo {
+            assignees(first: {NODE_SIZE}) {{
+            pageInfo {{
                 hasNextPage
                 endCursor
-            }
-            nodes {
+            }}
+            nodes {{
                 login
-            }
-            }
-            labels(first: 100) {
-            pageInfo {
+            }}
+            }}
+            labels(first: {NODE_SIZE}) {{
+            pageInfo {{
                 hasNextPage
                 endCursor
-            }
-            nodes {
+            }}
+            nodes {{
                 name
                 description
-            }
-            }
-            comments(first: 100) {
-            pageInfo {
+            }}
+            }}
+            comments(first: {NODE_SIZE}) {{
+            pageInfo {{
                 hasNextPage
                 endCursor
-            }
-            nodes {
-                author {
+            }}
+            nodes {{
+                author {{
                 login
-                }
+                }}
                 body
-            }
-            }
-        }
-        }
-    }
-    }
+            }}
+            }}
+        }}
+        }}
+    }}
+    }}
     """
     COMMENT_QUERY = """
     query ($owner: String!, $name: String!, $number: Int!, $cursor: String) {{
     repository(owner: $owner, name: $name) {{
         {object_type}(number: $number) {{
-        comments(first: 100, after: $cursor) {{
+        comments(first: {node_size}, after: $cursor) {{
```

Comment:
Hard to follow in this much diff, but should this be `NODE_SIZE` (capitalized)?

Reply:
we will be using `object_type` and `node_size` in format string later so, I kept this variable name in smaller case and in other queries we are using f-string and directly assigning the `NODE_SIZE`. Same for `LABELS_QUERY` and `ASSIGNEES_QUERY`.

Reply:
Is it ever variable? Or are you always just passing that constant as the variable to the format?

Reply:
Yes we are passing constant in format because above query is not f-string as we also needs pass `object_type`.